scripts/release-wrapper

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -ex
+
+if [ $# -ne 1 ]; then
+  echo "USAGE: $0 type"
+  exit 1
+fi
+
+TYPE=$1
+
+do_release () {
+  # TODO: add other release types
+  if [ "$TYPE" == "deb" ]; then
+    yarn run release $1
+    ./scripts/release-deb $1
+  fi
+}
+
+if [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
@dickeyxxx went another direction and made both `arch` and `branch` optional flags.  This gives me the flexibility I need for windows builds while at the same time keeping current behavior without having to refactor things now.